### PR TITLE
New services

### PR DESCRIPTION
**Pull request by emerham**
Changing the front end to use svg icons from font awesome library over the media system that we created with drupal. Merge after backend.

Closes DD-699

### CODE REVIEW

File: src/features/resources/ResourcesList.tsx [hunk 1]
```diff
@@ -43,9 +62,18 @@ const ResourceName = styled.div`
 const ResourceImg = styled.img`
   width: 3rem;
 `;
-
-const ResourceIcon = styled(Icon)`
-  height: auto;
-`;
-
+const IconLookupCustom = iconName => {
```
Comment (revgum) [inline src/features/resources/ResourcesList.tsx]: Can we move `logoMapping` and `IconLookupCustom` to a central place and make the two components import and use the same code? Maybe like `src/features/resources/resources-utils.tsx` or something along those things?

File: src/features/ResourcesCard.tsx [hunk 2]
```diff
@@ -99,4 +120,18 @@ const ResourcesCard: FC<{ categ: string; icon: IconDefinition }> = ({ categ, ico
 // Todo: Replace with actual empty state when ready in mockups.
 const EmptyState = () => <span>No resources available.</span>;
 
+const IconLookupCustom = iconName => {
```
Comment (revgum) [inline src/features/ResourcesCard.tsx]: Same comment regarding moving this code to one spot..

**Comment by mxcosu** [comment]
This should be ready to me merged. Code review

File: src/features/ResourcesCard.tsx (hunk 2 above)
Comment (mxcosu) [reply src/features/ResourcesCard.tsx]: Done

**Pull request merged**